lib/rx/middleware/response/rx_parser.rb

```diff
@@ -13,7 +13,7 @@ class RxParser < Faraday::Response::Middleware
         # @return [Faraday::Env]
         #
         def on_complete(env)
-          Rails.logger.info('RX Parser Debugging', body: env.body, status: env.status)
+          Rails.logger.info('RX Parser Debugging', status: env.status, body: env.body)
```

Comment:
in some logs, the character limit is reached and not showing the status. Switching the two so we get it first.